### DISCUSSION TITLE
Add Syndicate Seals to the Chameleon mechanic

### DISCUSSION THREAD

**Discussion by IlyaS-1243** [space-wizards/space-station-14#31315] [Feature Request]
Hello, I propose to Change the Syndicate Seal for Syndicate Agents. The meaning of this item, I think, is already clear: Allows you to put any seal on documents. This will make the Syndicate seal useful and not just a trinket. Why Balance? Allows you to deceive the station crew and forge documents, but because of its appearance, Enemy Equipment is immediately visible and the below-mentioned Corrections make it Balanced. 
Balance:
1.Add CD or Charges like Emag 
2.Increase the price in Uplink, for example: 3-4 TK 
3. Leave the appearance as Default, thereby making it clear to everyone that the Seal is Enemy Equipment.

**Comment by IlyaS-1243** [comment]
Of course, the Seal will not be able to parody the Important Seal of the Type: Captain and Central Command

**Comment by nymhole** [comment]
chameleon is visual only, the stamps would require physical morphing, the chameleon seal would be attached to the paper and able to be removed from the paper making it metagamabkle

**Reply by nymhole** [reply]
this would also mean that printing it should make a paper with a real stamp, im not sure how this work.
